src/it/resources/parameters-template.txt

```diff
@@ -40,7 +40,7 @@ paths:
             "adProofsRoot": "a84f62a669fb3684308ea609af6cd831b939b70210307e143a05321ce8efeda2",
             "transactionsRoot": "cc32add0ada11b6a81f07b61a6c606b2277af9999b453538d32b3c409630bce1",
             "extensionHash": "0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8",
-            "equihashSolutions": "00000914000125c300013c440001c51e00000fbb00002a960000edc40001baee000065d500006ff90000e43f00011ca500016f270001781d00017b150001d54d0000493500012c19000171eb000184f50000720d0000e0f80000bcd9000181bc00004dc40000ef250000d09300011c540000f1190001d6f60000ffaf0001104f",
+            "powSolutions": "???",
```

Comment:
Let's put it here instead of `???`
```
"powSolutions" : {
    "pk" : "0350e25cee8562697d55275c96bb01b34228f9bd68fd9933f2a25ff195526864f5",
    "w" : "032e3f5edb88f3cc7384bfedc892fc8dcb10d7a3bf3741d08a62cc701848d0932c",
    "n" : "0000000000000000",
    "d" : 603083798111021851164432213586916186738093029948325633833495911523787854249
},
```